cmd/repl/repl.go

```diff
@@ -269,8 +264,8 @@ func main() {
 	}
 	scanner := bufio.NewScanner(reader)
 	if isTerminal {
-		if !commandExists("it2check") {
-			fmt.Println("Warning: defaults to non-sixel mode")
+		if !it2Check {
+			fmt.Println("Mode: non-sixel")
```

Comment:
Why the warning message is not `cannot found it2check toolkit`?

Reply:
> Why the warning message is not `cannot found it2check toolkit`?

I believe that code was somewhat temporary that is committed accidentally.
https://github.com/sql-machine-learning/sqlflow/pull/1507/commits/c9388694ef22363f16659fb0d7456d2de8514f92